tests/unit/amo/reducers/test_home.js

```diff
@@ -372,10 +372,24 @@ describe(__filename, () => {
           },
         },
         secondary: {
-          cta: heroShelves.secondary.cta,
-          description: heroShelves.secondary.description,
-          headline: heroShelves.secondary.headline,
-          modules: heroShelves.secondary.modules,
+          cta: {
+            url: heroShelves.secondary.cta.url,
+            outgoing: heroShelves.secondary.cta.outgoing,
+            text: heroShelves.secondary.cta.text[lang],
+          },
+          description: heroShelves.secondary.description[lang],
+          headline: heroShelves.secondary.headline[lang],
+          modules: heroShelves.secondary.modules.map((module) => {
```

Comment:
same here, I think it's fine to keep the `map()` if we don't have a `createInternalXXX()` for `modules`, but we could use `createInternalSecondaryHeroModule()`.

Reply:
https://github.com/mozilla/addons-frontend/pull/10086/commits/7d92b37c782685e84da506439bf1f1cf070b5806